app/api/v1/models/ballot.py

```diff
@@ -1,18 +1,23 @@
-from typing import Any, List, Optional
+from typing import Any, Dict, List, Optional
 
 from .base import Base
 from .election import ElectionDescription, CiphertextElectionContext
+from .guardian import Guardian, GuardianId
 
 
 __all__ = [
+    "AcceptBallotRequest",
```

Comment:
I was genuine unsure of whether to alphabetize of not, but i'm in support of this.